src/MetalPerformanceShaders/MPSDefs.cs

```diff
@@ -423,7 +457,8 @@ public enum MPSNNComparisonType : ulong {
 		GreaterOrEqual,
 	}
 
-	[TV (11, 3), Mac (10, 13, 4), iOS (11, 3)]
+	[Introduced (PlatformName.MacCatalyst, 13, 0)]
+	[TV (11,3), Mac (10,13,4), iOS (11,3)]
```

Comment:
Reduce whitespace noise:

```suggestion
	[TV (11, 3), Mac (10, 13, 4), iOS (11, 3)]
```